src/platform/nrf_pca10001/lib.rs

```diff
@@ -2,16 +2,35 @@
 #![crate_type = "rlib"]
```

Comment:
There are no changes here that are related to the UART, and many of the changes are incorrectly reverting previous changes. Just leave this file as is.